public/app/plugins/datasource/cloudwatch/dataquery.cue

```diff
@@ -0,0 +1,172 @@
+// Copyright 2023 Grafana Labs
+//
+// Licensed under the Apache License, Version 2.0 (the "License")
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
I'm not a licensing expert, but why would this file be under a different license than the rest of grafana?

Reply:
This is probably a question for @sdboyer.